assets/dev/js/editor/components/settings/page/component.js

```diff
@@ -1,4 +1,4 @@
-import ComponentBase from 'elementor-api/modules/component-base';
+import ComponentBase from 'elementor-editor/component-base';
```

Comment:
This specific solution is good for fixing the bug in the "page-settings" only, I don't like it and I think a better solution should be found.